app-modules/engagement/resources/views/components/engagement-timeline-item.blade.php

```diff
@@ -98,7 +98,7 @@ class="my-4 rounded-lg border-2 border-gray-200 p-2 text-base font-normal text-g
         <div class="flex flex-col">
             <p class="text-xs text-gray-400 dark:text-gray-500">Body:</p>
             <div class="prose dark:prose-invert">
-                {{ str($record->getBody())->markdown()->sanitizeHtml()->toHtmlString() }}
+                {{ $record->getBody() }}
```

Comment:
We are removing a `sanitizeHtml` here.

I just want to confirm whether or not output sanitization is covered upstream or not if we do this?

Reply:
It should be good now.